src/bot.py

```diff
@@ -198,18 +198,18 @@ def get_turn_points(self, train_idx, target_point, point_from=None):
         :param train_idx: int - train index
         :param target_point: int - target point index
         :param point_from: int - point index to build way from, if None - the way builds from current point
-        :return: list - list of turn points
+        :return: 2-tuple of list and int. List - list of turn points. Int - distance to target point
```

Comment:
:return: 2-tuple where the first item is a trip length to the target point and the second item is a list of turn points